README.md

```diff
@@ -1,48 +1,21 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
-## Getting Started
+## Vet clinic database
+In the project projects, I use a relational database to create the data structure for a vet clinic.
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to use the same text format to customize your README.md file to look more professional. You need to copy the raw data of the README template instead of just copying the text itself.